shadowapi/src/main/java/org/robolectric/shadow/api/Shadow.java

```diff
@@ -33,7 +33,7 @@ public static Object newInstanceOf(String className) {
     try {
       Class<?> aClass = Shadow.class.getClassLoader().loadClass(className);
       return SHADOW_IMPL.newInstanceOf(aClass);
-    } catch (ClassNotFoundException e) {
+    } catch (ClassNotFoundException | RuntimeException e) {
```

Comment:
I am not sure it is good to catch a generic RuntimeException here. For places like this it is good to have a verbose exception so people understand what is going on. This can swallow a lot of potential exceptions, making it hard to debug. Why not just fix the root cause of the issue?

Reply:
I will investigate the origin issue again this weekend to check whether we can have better solution for it.